src/utils/batchWarnings.js

```diff
@@ -1,6 +1,6 @@
 import chalk from 'chalk';
-import { stderr } from '../logging.js';
-import relativeId from '../../../src/utils/relativeId.js';
+import { stderr } from '../../bin/src/logging.js';
```

Comment:
This statement in `src/` imports from `bin/`. Maybe it's possible to pass in `stderr`?